src/js/components/index.jsx

```diff
@@ -1,7 +1,8 @@
 var ReactDOM = require('react-dom'),
-    Sidebars = require('./Sidebars.jsx');
+    Sidebars = require('./Sidebars.jsx'),
+    React = require('react');
 
 module.exports = ReactDOM.render(
   <Sidebars />,
```

Comment:
Anywhere we used JSX we must explicitly require React `var React = require('react');`, because the React loader considers using a component via JSX syntax is equivalent to calling React's method to create that element.